src/data-model-providers/codegen/CodegenDataModelProvider_Write.cpp

```diff
@@ -123,7 +130,7 @@ DataModel::ActionReturnStatus CodegenDataModelProvider::WriteAttribute(const Dat
         }
     }
 
-    ContextAttributesChangeListener change_listener(CurrentContext());
+    ContextAttributesChangeListener change_listener(mContext);
```

Comment:
Shouldn't we just return error if we land here before Startup was called (i.e. if there is no mContext), and then leave ContextAttributesChangeListener the way it was before?